files/bootstrap.sh

```diff
@@ -123,6 +124,11 @@ while [[ $# -gt 0 ]]; do
             shift
             shift
             ;;
+        --imds-host)
```

Comment:
you should propogate this down to `max-pods-calculator.sh`, right? https://github.com/awslabs/amazon-eks-ami/pull/931/files#diff-049390d14bc3ea2d7882ff0f108e2802ad9b043336c5fa637e93581d9a7fdfc2R519

Reply:
ah you're right, I missed this since I was setting the IMDS_HOST env var in the docker container which was adding it to both scripts. Fixed